packages/rancher-vsphere-cpi/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,12 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -11,6 +11,9 @@
+ 
+ cloudControllerManager:
+   image:
++    "override":
```

Comment:
This looks like it'll override the image for all Kubernetes versions. Would it make more sense to just define this as the image for 1.23 clusters and let older versions still use whatever tag they would normally use?

Reply:
I am going to defer that to @PennyScissors if that is what we should consider as an approach.

Reply:
That's correct, this will override for all Kubernetes versions. If you want 1.23 specifically to use a 1.22 image, you could try changing "override" for "1.23", and leave the repo and tag with the 1.22 images as is.

Reply:
@briandowns @luthermonson @rosskirkpat any opinions?

Reply:
Let's go with specifying this as the version for 1.23 clusters only.

Reply:
Ok, I will get it changed and pushed